amanjaiman1/Product_3D
# feat(customizer): implemented core

## Pull Request Description

This pull request adds the functionality for zoom in and zoom out options to the UI component of the customizer.

### Changes Made

- Implemented separate handle functions for zoom in and zoom out.
- Used Three.js methods to achieve the zoom effect.
- Updated the UI components or event listeners to trigger the respective handle functions.

### How to Test

1. Clone the repository and switch to the branch for this PR.
2. Start the application and navigate to the customizer screen.
3. Use the provided UI components for zoom in and zoom out.
4. Verify that the zoom functionality works as expected, adjusting the view accordingly.

### Related Issue

#315
